hulks/check_logger.py

```diff
@@ -17,6 +17,9 @@ def validate(self, filename, **options):
             if 'getLogger(' not in line:
                 continue
 
+            if line.startswith('    '):
```

Comment:
Maybe change this to be more generic and accept `8` spaces and tabs?

Reply:
maybe just 1+ whatever blank char